addons/event_sale/models/event_registration.py

```diff
@@ -31,13 +32,13 @@ def _check_ticket_seats_limit(self):
             if record.event_ticket_id.seats_max and record.event_ticket_id.seats_available < 0:
                 raise ValidationError(_('No more available seats for this ticket'))
 
-    def _check_auto_confirmation(self):
```

Comment:
Why removing this ?

Reply:
When the auto_confirm flag is set on the event the registration should always be set on confirmed

Reply:
Ok.